cmake/3rdParty/Platform/iOS/BuiltInPackages_ios.cmake

```diff
@@ -31,3 +30,10 @@ ly_associate_package(PACKAGE_NAME libsamplerate-0.2.1-rev2-ios   TARGETS libsamp
 ly_associate_package(PACKAGE_NAME OpenSSL-1.1.1o-rev1-ios        TARGETS OpenSSL         PACKAGE_HASH 1325bbb2dcc97fe33d8db71e4b0a61e95e9e77558e3ce6767a1b5d330240f78f)
 ly_associate_package(PACKAGE_NAME zlib-1.2.11-rev5-ios           TARGETS ZLIB            PACKAGE_HASH c7f10b4d0fe63192054d926f53b08e852cdf472bc2b18e2f7be5aecac1869f7f)
 ly_associate_package(PACKAGE_NAME lz4-1.9.3-vcpkg-rev4-ios       TARGETS lz4             PACKAGE_HASH 588ea05739caa9231a9a17a1e8cf64c5b9a265e16528bc05420af7e2534e86c1)
+
+set(AZ_USE_PHYSX5 OFF CACHE BOOL "When ON PhysX Gem will use PhysX 5 SDK")
+if(AZ_USE_PHYSX5)
+    ly_associate_package(PACKAGE_NAME PhysX-5.1.1-rev1-ios           TARGETS PhysX       PACKAGE_HASH 3495e0bc1404a369fb6f4880ffc525c0a9d860867f5c524b33887cfd08f5c376)
+else()
+    ly_associate_package(PACKAGE_NAME PhysX-4.1.2.29882248-rev5-ios  TARGETS PhysX       PACKAGE_HASH 4a5e38b385837248590018eb133444b4e440190414e6756191200a10c8fa5615)
+endif()
```

Comment:
These ly_associate_package calls can be moved to the PhysX gem itself, so that if the PhysX gem isn't part of the project, it can skip downloading the libraries.

Realistically though, my suggested change will not make a difference since the PhysX gem is part of the engine via the `engine.json`, but if the gem was ever not associated with the engine, but instead a project, then it would become applicable

Reply:
That's a good idea, when it comes to supporting multiple default physics backends we won't have to have PhysX enabled. 
At the moment we can have more than one physics engine simulating, but the actual API backend is always PhysX.